README.md

```diff
@@ -1,28 +1,130 @@
-# AWS libcrypto (AWS-LC) Public Preview
+# AWS libcrypto (AWS-LC)
 
 [![Join the chat at https://gitter.im/awslabs/aws-lc](https://badges.gitter.im/awslabs/aws-lc.svg)](https://gitter.im/awslabs/aws-lc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-AWS libcrypto (AWS-LC) is a general-purpose cryptographic library maintained by
-the AWS Cryptography team for use with AWS Nitro Enclaves and other AWS applications. 
-AWS-LC іs based on code from the Google BoringSSL project and the OpenSSL project.
+AWS-LC is a general-purpose cryptographic library maintained by the AWS Cryptography
+team for AWS and there customers.It іs based on code from the Google BoringSSL project
+and the OpenSSL project.
+
+AWS-LC contains portable C implementations of algorithms needed for TLS and common
+applications. For performance critical algorithms, optimized assembly versions are
```

Comment:
Do we want to include which algorithms we have optimized assembly versions for?

Reply:
I want to include that change with the performance updates change.